pkg/k8shandler/status.go

```diff
@@ -373,15 +373,15 @@ func (er *ElasticsearchRequest) updatePodNodeConditions(status *api.Elasticsearc
 				if podCondition.Type == v1.PodScheduled && podCondition.Status == v1.ConditionFalse {
 					podCondition.Type = v1.PodReasonUnschedulable
 					podCondition.Status = v1.ConditionTrue
-					updatePodUnschedulableCondition(&nodeStatus, podCondition)
+					updatePodUnschedulableCondition(&status.Nodes[nodeIndex], podCondition)
```

Comment:
is this the same as `node`?

Reply:
yes and no... as a getter it resolves to the same object, but since we're trying to update it as we iterate over it the change doesn't persist to `status.Nodes`